gpgme.go

```diff
@@ -546,6 +548,18 @@ func (c *Context) Encrypt(recipients []*Key, flags EncryptFlag, plaintext, ciphe
 	return handleError(err)
 }
 
+func (c *Context) SetPassphrase(passphrase string) {
+	logrus.Debugf("Setting GPGME passphrase callback", passphrase)
+	callback := C.gpgme_passphrase_cb_t(C.passphrase_cb)
+	cPass := C.CString(passphrase)
+	C.gpgme_set_pinentry_mode(c.ctx, C.GPGME_PINENTRY_MODE_LOOPBACK);
```

Comment:
This won’t compile with the GPGME 1.3.2 version this repo is currently targeting.

We should almost certainly just give up on that version, and move on, sure. If I’m reading https://access.redhat.com/downloads/content/package-browser correctly, neither of the changes in this repo are necessary for RHEL ≥ 8.